selenium/src/main/java/com/deque/html/axecore/extensions/WebDriverExtensions.java

```diff
@@ -35,7 +35,7 @@ private WebDriverExtensions() {}
    * @throws IOException if analysis of page reaches an error
    * @throws OperationNotSupportedException if an unaccepted error occurs
    */
-  public static Results analyze(final WebDriver webDriver)
+  public static AxeResults analyze(final WebDriver webDriver)

```

Comment:
All instances of `AxeResults` will need to be reverted back to `Results`

Reply:
Reverted all instances of Axe Results to Results in selenium module